src/view/draggable/connected-draggable.js

```diff
@@ -263,12 +264,19 @@ const mapDispatchToProps: DispatchProps = {
 // Leaning heavily on the default shallow equality checking
 // that `connect` provides.
 // It avoids needing to do it own within `Draggable`
-export default connect(
+const ConnectedDraggable: OwnProps => Node = (connect(
   // returning a function to ensure each
   // Draggable gets its own selector
-  makeMapStateToProps,
-  mapDispatchToProps,
+  (makeMapStateToProps: any),
+  (mapDispatchToProps: any),
   null,
   { storeKey },
-)(Draggable);
+): any)(Draggable);
```

Comment:
this `any`'s is the result of incomplete react-redux definitions. I'm thinking about solving this problem later.